collections/_pages/gdpr.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single-section-page
+title: Privacy policy
+slug: gdpr
+---
+{:.lead}
+Created in compliance with applicable legislation and Regulation (EU) 2016/679 of the European Parliament and Council on the protection of natural persons with regard to the processing of personal data and on the free movement of such data. This Privacy Policy is valid as of 6 October 2020.
+
+## Data processor
+
+Your personal data is processed by Otevřená data o klimatu, z. ú., registered at the Regional Court in Brno, file number U 300.
+
+Address: Cyrilská 7, 602 00 Brno, ID: 095 55 587
+
+Contact person: Anna Macková 
```

Comment:
Is this still true?
(I know she's listed on the Czech version as well)

Do we have to mention a contact person?

Reply:
I can't tell if the name is still up to date, but I would keep it consistent with the Czech version. We can try to find out if a different name should be put there.

Yes, [contact info is mandatory](https://gdpr.eu/privacy-notice/):
> If an organization is collecting information from an individual directly, it must include the following information in its privacy notice:
> * The identity and contact details of the organization, its representative, and its Data Protection Officer
> […]
> If an organization obtains your data indirectly (via another organization) its privacy notice must provide all the same information

Reply:
Ack